### ISSUE TITLE
Verbose logging even when log level is info

### ISSUE THREAD

**Issue by rogerjak** [getgauge/gauge-dotnet#135]
Since 0.3.0 the console output and log file is filled with these:
`Microsoft.AspNetCore.Routing.EndpointMiddleware[0]`
`Microsoft.AspNetCore.Hosting.Diagnostics[1]`
For each one there are a couple of more rows belonging to them.
I don't think they should be there for log level info, only debug.

**Comment by sriv** [comment]
thanks for raising this. gauge-dotnet 0.3.0 uses a pure c# gRPC implementation, which in turn uses the asp-net core libs to host the server. 

Raised a fix for this.

**Comment by sriv** [comment]
Just realized that #132 is also related.

**Comment by raweaver** [comment]
I've noticed similar behavior with gauge-dotnet 0.3.1 where the Microsoft.AspNetCore info messages appear in the console output on a normal run.

```
# Specification Heading
info
: Microsoft.AspNetCore.Hosting.Diagnostics[1]
Request starting HTTP/2 POST http://127.0.0.1:54594/gauge.messages.Runner/InitializeSpecDataStore application/grpc
info: Microsoft.AspNetCore.Routing.EndpointMiddleware[0]
Executing endpoint 'gRPC - /gauge.messages.Runner/InitializeSpecDataStore'
info
: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]
Executed endpoint 'gRPC - /gauge.messages.Runner/InitializeSpecDataStore'
info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
Request finished in 1.6942ms 200 application/grpc
info
: Microsoft.AspNetCore.Hosting.Diagnostics[1]
Request starting HTTP/2 POST http://127.0.0.1:54594/gauge.messages.Runner/StartSpecExecution application/grpc
```

I then set GAUGE_LOG_LEVEL = DEBUG in my environment and the Microsoft.AspNetCore logging statements did not appear in the console output. This is counterintuitive that the additional logging output is suppressed when enabling debug logging level.

I would appreciate any help in resolving this.

Thank you!

**Comment by zabil** [comment]
This should be fixed in 0.3.1 via 

https://github.com/getgauge/gauge-dotnet/pull/133/files#diff-280ecf7c70e274bad023c9acea61103fae54640ec4fdc58b35dd6ff40d199e74L41-L43

Can you please the dotnet plugin is using the latest version?

**Comment by raweaver** [comment]
@zabil Thank you for the quick response. You are correct, this is fixed in 0.3.1. My apologies. I thought I was running 0.3.1 because I ran gauge update dotnet, but the update had installed 0.3.0.

How does 0.3.1 become the version installed when gauge checks for plugin updates?

Thank you!

**Comment by zabil** [comment]
Ok that explains it and thanks for pointing it out. For some reason the latest plugin version is not updated here

https://github.com/getgauge/gauge-repository/blob/master/dotnet-install.json#L4

Need to check that and will push a fix soon.

**Comment by rogerjak** [comment]
> Ok that explains it and thanks for pointing it out. For some reason the latest plugin version is not updated here
> 
> https://github.com/getgauge/gauge-repository/blob/master/dotnet-install.json#L4
> 
> Need to check that and will push a fix soon.

Gauge 1.1.7 still won't update dotnet to 0.3.1... :(

**Comment by zabil** [comment]
0.3.1 is available at https://github.com/getgauge/gauge-dotnet/releases/tag/v0.3.1

However it's not updated at https://github.com/getgauge/gauge-repository/blob/fa99e41a4afd6a03f21f4ec456aeb789f6352bfc/dotnet-install.json#L4

Please feel free to send a pull request by copying the section related to 0.3.0 and bumping up the version and the download urls to 0.3.1. You just need to make sure that it's a valid json file.

**Comment by sriv** [comment]
gauge-dotnet 0.3.1 is added to the repository via https://github.com/getgauge/gauge-repository/pull/63

Am closing this issue, please feel free to comment if this isn't fixed in 0.3.1.
